src/cljs/athens/keybindings.cljs

```diff
@@ -326,6 +327,14 @@
     ;;(= key-code KeyCodes.CLOSE_SQUARE_BRACKET)
 
 
+;; This should probably reside in another file as it has nothing to do
```

Comment:
I think you could move this function `db.cljs` since that's where we're storing a lot of common functionality.

Reply:
I was wondering where would be the best place. This function is about updating the block's atom, not the datascript db, but if that's not a problem I'll move it to db.cljs.

Reply:
I think it's fine to refactor the logic into a function, but I would keep it within `keybindings` for now and not `db` for the reasons you state @bonobo-d